communications/newsletters/newsletter_33_Apr2022.md

```diff
@@ -0,0 +1,259 @@
+[Sign up on the Eventbrite](https://www.eventbrite.co.uk/e/fireside-chat-making-virtual-participation-a-global-default-tbc-tickets-318719758377).
+
+## Sign up for our Book Dash 📚💨
+
+**The 7th [Book Dash](https://the-turing-way.netlify.app/community-handbook/bookdash.html) event will take place online from 16 to 20 May 2022.** Submit your application via this [Google form](https://forms.gle/uEbu1FJMmdfgGwvW6).
+The Deadline for submission is 21 April 2022.
+Please reach email me at [asteele@turing.ac.uk](mailto:asteele@turing.ac.uk) if you think you'll need need more time to complete the application.
+
+We invite applications from members of our community, including both new and existing contributors. Our attendees bring some understanding of where they can contribute and collaboratively improve _The Turing Way_ as a whole.
+If participating in _The Turing Way_ community for the first time, we encourage you to first participate in a Collaboration Café or other community events shared in this newsletter (also see: [bit.ly/turingway](http://bit.ly/turingway)).
+
+This application will take 20-30 minutes to complete.
+You can use the [template document](https://tinyurl.com/tw-bookdash-template) to prepare your draft.
+If you'd like to work with someone on drafting your application, please join us at Collaboration Cafe.
+Don't hesitate to reach out – we're here to give you the support you need.
+
+Don't forget that Book Dashes usually end with a [Community Share Out and Contributor Celebrations Event](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-events.html#community-share-outs-contributor-celebration), which you can sign up for on [Eventbrite](https://www.eventbrite.co.uk/e/the-turing-way-community-share-out-and-contributor-celebrations-may-2022-tickets-323373307267).
+
+## Join us at Collaboration Cafe ☕
+
+**The next Collaboration Café will is TODAY on 20 April 2022 15:00 BST (UTC+1).**
+We will host a room on pre-applications discussions for the upcoming Book Dash.
+
+Please find details and sign up here: [hackmd.io/@turingway/collaboration-cafe](https://hackmd.io/@turingway/collaboration-cafe).
+You can also sign up to host one of the future Collaboration Cafés on 4 May and 18 May 2022. We hope to see you there!
+
+---
+
+## Community News
+
+### Featured: UK's Goldacre review!
+
+![Alt:Screenshot of Goldacre report page and title](https://i.imgur.com/RFFH8iP.jpg)
+_The Goldacre Review was published on 7 April 2022._
+
+We are thrilled and honoured that [_The Turing Way_ was recently featured in the Goldacre Review](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis).
+You can read the [full report online](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis).
```

Comment:
Noting that Esther flagged that should probably only use one link as these are both the same...

Reply:
Maybe would be worth writing one sentence on why the Turing Way was highlighted?

Reply:
```suggestion
_The Turing Way_ was highlighted in the context of addressing *a range of specific challenges around open working, the limits of open code, and the strong positive relationship between open working methods and innovative commercial activity in health data science*.
You can read the [full report online](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis/better-broader-safer-using-health-data-for-research-and-analysis).
```

Reply:
Great idea Arielle, added a sentence.